jumpscale/sals/reservation_chatflow/deployer.py

```diff
@@ -534,7 +534,7 @@ def check_pool_capacity(self, pool, cu=None, su=None):
 
     def ask_name(self, bot):
         name = bot.string_ask(
-            "Please enter a name for you workload (Can be used to prepare domain for you and needed to track your solution on the grid )",
+            "Please enter a name for your workload (Can be used to prepare domain for you and needed to track your solution on the grid )",
```

Comment:
There shouldn't be a space before the closing parenthesis

Reply:
updated in https://github.com/threefoldtech/js-sdk/pull/975/commits/f3210f4e8a0f8dec94f56edbcbd181fce47d7cef